package.json

```diff
@@ -46,7 +51,8 @@
     "harmony-test": "node --harmony --harmony_arrow_functions ./node_modules/tape/bin/tape test/*.js",
     "rebuild": "rm -rf node_modules && npm run setup",
     "samples": "./bin/index -f examples",
-    "symlink": "ln -s $PWD/bin/index /usr/local/bin/fly"
+    "symlink": "ln -s $PWD/bin/index /usr/local/bin/fly",
+    "prepublish": "npm run build"
```

Comment:
I'll try to simplify the scripts a bit, but this is a nice contribution. Thanks!